packages/message-box-react/src/MessageBox.tsx

```diff
@@ -6,20 +6,60 @@ interface Props {
     children: ReactNode;
     fullWidth?: boolean;
     className?: string;
+    inverted?: boolean;
 }
 
 type messageTypes = "info" | "error" | "success" | "warning";
 
 function messageFactory(messageType: messageTypes) {
-    return function messageBox({ title, fullWidth, className = "", children }: Props) {
+    return function messageBox({ title, fullWidth, className = "", children, inverted }: Props) {
         const componentClassName = classNames("jkl-message-box", "jkl-message-box--" + messageType, className, {
             "jkl-message-box--full": fullWidth,
+            [`jkl-message-box--${messageType}--dark`]: inverted,
         });
 
+        const getIcon = (type: string) => {
```

Comment:
```suggestion
        const getIcon = (type: messagesTypes) => {
```

Reply:
Mulig du kan droppe default klausulen også. Eller så gjelder det for enums bare. Kan testes.

Ser gjerne vi bruker return isteden for break også, så slipper du å definere ikonet icon først også